addons/base_import_module/controllers/main.py

```diff
@@ -37,7 +37,7 @@ def login(self, login, password, db=None):
             'X-CSRF-TOKEN': request.csrf_token(),
         })
 
-    @route('/base_import_module/upload', type='http', auth='user', methods=['POST'])
+    @route('/base_import_module/upload', type='http', auth='user', methods=['POST'], csrf=False)
```

Comment:
I'm not sure that removing a security layer is in the scope of the migration. Or is it to avoid something ? I hope not. I would be very angry. And when I'm angry I want to break things.